testing/bcc/APKBUILD

```diff
@@ -1,17 +1,34 @@
 # Maintainer: Adam Jensen <acjensen@gmail.com>
 pkgname=bcc
-pkgver=0.8.0
+pkgver=0.9.0
 pkgrel=0
 pkgdesc="A toolkit for creating efficient kernel tracing and manipulation programs"
 url="https://github.com/iovisor/bcc/"
-arch="aarch64 x86_64"
+arch="aarch64 x86 x86_64"
 license="Apache-2.0"
 # bcc's test suite requires privileged access to run BPF programs
 options="!check"
 depends="python"
 makedepends="tar git llvm7-dev llvm7-static clang-dev clang-static cmake flex-dev
 	bison luajit-dev build-base iperf linux-headers elfutils-dev zlib-dev"
-source="$pkgname-$pkgver.tar.gz::https://github.com/iovisor/$pkgname/archive/v$pkgver.tar.gz"
+source="$pkgname-$pkgver.tar.gz::https://github.com/iovisor/$pkgname/archive/v$pkgver.tar.gz
+	10-bcc-include-stddef.patch
+	20-libbpf-include-stddef.patch"
+
+prepare() {
+	cd "$builddir"
+	rm -rf tests/* && touch tests/CMakeLists.txt
+
+	# libbpf does not currently have tagged releases, so we fetch them here
+	# before the patches are applied.
+	mkdir -p "$builddir/build/src/cc"
+	cd "$builddir/src/cc"
+	git clone https://github.com/libbpf/libbpf.git
```

Comment:
it should be separate package based on tags or date

Reply:
Done. Thanks!